### ISSUE TITLE
Button BF accessibility documentation

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#770]
The current BF accessibility documentation is not ready in implementation, although some features are built in native (text size) the overall native / web implementation of this component lacks these features.

https://brandfactory.telefonica.com/document/1846#/components/buttons/accesibility-1

- [x] Remove accessibility section until the [a11y project](https://github.com/Telefonica/mistica-design/issues/557) is completed and buttons are included.
